code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -151,6 +151,12 @@
 	icon_state = "round_bottle"
 	list_reagents = list("sulfonal" = 30)
 
+/obj/item/reagent_containers/glass/bottle/antiviral
+	name = "Spaceacillin Bottle"
+	desc = "A small bottle containing a commonly used medical antidote."
```

Comment:
I'm not sure antidote is the right word here. You're not using it to neutralize a poison.

Reply:
Ooh, right, what would it be instead?

Reply:
Maybe "A small bottle containing a commonly used medicine." ?

Reply:
Hm that could work I guess. I'll change that up and await some more feedback.

Reply:
Well something went terribly wrong. Apparently I have over 100k removes which is more than insanely unrealistic. Gotta have to look into that.

Reply:
Did you run the mapmerge scripts?